.github/workflows/connectors_pre_release_checks.yml

```diff
@@ -51,4 +53,4 @@ jobs:
           github_token: ${{ env.PAT }}
           s3_build_cache_access_key_id: ${{ secrets.SELF_RUNNER_AWS_ACCESS_KEY_ID }}
           s3_build_cache_secret_key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
-          subcommand: "connectors --modified test --only-step=version_inc_check --global-status-check-context='Version increment check for Java connectors' --global-status-check-description='Checking if java connectors modified in this PR got their version bumped'"
+          subcommand: "connectors --modified test --only-step=version_inc_check --only-step=qa_checks --global-status-check-context='Connectors Pre-Release Check' --global-status-check-description='Checking if connectors modified in this PR are ready for release'"
```

Comment:
This looks right to me, but putting this down explicitly: I don't see that you are filtering Java / Python connectors, but I think this is OK because previously this ran for all connectors anyway.

Reply:
Yes, that's correct. 👍 

This is a case where Java team was trying to optimize their process and inadvertently created side-effects for Python connectors also. My update basically extends what they started and applies (intentionally) across all connectors.